src/xrGame/inventory_item.cpp

```diff
@@ -87,6 +87,8 @@ CInventoryItem::~CInventoryItem()
 #endif // #ifndef MASTER_GOLD
 }
 
+extern int g_normalize_mouse_sens;
```

Comment:
Please, place it to the beginning of the file: after `DESCRIPTION_KEY` and before `CInventoryItem::NetSync`